From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 20 May 2019 00:53:57 +0200
Subject: [PATCH] {NOT UPSTREAMABLE} hwtracing/intel_th: make sure i is used

Building with non-x86 arch fails becase i is only used if CONFIG_X86.
---
 drivers/hwtracing/intel_th/msu.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/hwtracing/intel_th/msu.c b/drivers/hwtracing/intel_th/msu.c
index 81bb54fa3ce8..2163d5749390 100644
--- a/drivers/hwtracing/intel_th/msu.c
+++ b/drivers/hwtracing/intel_th/msu.c
@@ -815,6 +815,8 @@ static int msc_buffer_win_alloc(struct msc *msc, unsigned int nr_blocks)
 	for (i = 0; i < ret; i++)
 		/* Set the page as uncached */
 		set_memory_uc((unsigned long)msc_win_block(win, i), 1);
+#else
+(void)&i;
 #endif
 
 	win->nr_blocks = ret;
@@ -874,6 +876,8 @@ static void msc_buffer_win_free(struct msc *msc, struct msc_window *win)
 	for (i = 0; i < win->nr_blocks; i++)
 		/* Reset the page to write-back */
 		set_memory_wb((unsigned long)msc_win_block(win, i), 1);
+#else
+(void)&i;
 #endif
 
 	__msc_buffer_win_free(msc, win);
-- 
